### ISSUE TITLE
zip Linux doesn't start

### ISSUE THREAD

**Issue by stefonarch** [nano-wallet-company/nano-wallet-desktop#20]
Hanging at the loading animation, creating config folder (88K) and that's it.
Arch Linuxx up to date
```
~/.config$ ls -lh Nano\ Wallet\ Desktop/
totale 20K
drwx------ 2 stef users 4,0K 21 giu 20.45  GPUCache
drwx------ 3 stef users 4,0K 21 giu 20.45 'Local Storage'
-rw-r--r-- 1 stef users  444 21 giu 20.46  log.log
-rw------- 1 stef users   69 21 giu 20.45  Preferences
-rw-r--r-- 1 stef users  142 21 giu 20.46  window-state.json
```
```
 cat Nano\ Wallet\ Desktop/log.log 
[2018-06-21 20:45:07.134] [info] Starting application
[2018-06-21 20:45:07.194] [info] Deleting outdated assets: config.json, rai_node
[2018-06-21 20:45:07.915] [info] Application window ready to show: Nano Wallet Desktop
[2018-06-21 20:46:01.271] [info] Starting application
[2018-06-21 20:46:01.324] [info] Deleting outdated assets: config.json, rai_node
[2018-06-21 20:46:02.068] [info] Application window ready to show: Nano Wallet Desktop
[2018-06-21 20:48:28.370] [info] Starting application
[2018-06-21 20:48:28.434] [info] Deleting outdated assets: config.json, rai_node
[2018-06-21 20:48:29.152] [info] Application window ready to show: Nano Wallet Desktop

```

**Comment by pentastream** [comment]
Same issue here. I've found the following in the console:

```
serve://dist/sw-registration-96562d3f04aac78eb85b38ec252eac9a.js:5 Service Worker registration failed with SecurityError: Failed to register a ServiceWorker: The URL protocol of the current origin ('serve://dist') is not supported.
vendor-8c88cdde22c88a7983ca101bd4964015.js:3332 Error while processing route: index Unexpected token < in JSON at position 0 SyntaxError: Unexpected token < in JSON at position 0
    at parse (<anonymous>)
    at y (serve://dist/assets/vendor-8c88cdde22c88a7983ca101bd4964015.js:4060:25)
    at S (serve://dist/assets/vendor-8c88cdde22c88a7983ca101bd4964015.js:4068:9)
    at O (serve://dist/assets/vendor-8c88cdde22c88a7983ca101bd4964015.js:4066:43)
    at e.invoke (serve://dist/assets/vendor-8c88cdde22c88a7983ca101bd4964015.js:2092:186)
    at e.flush (serve://dist/assets/vendor-8c88cdde22c88a7983ca101bd4964015.js:2085:147)
    at e.flush (serve://dist/assets/vendor-8c88cdde22c88a7983ca101bd4964015.js:2096:382)
    at e._end (serve://dist/assets/vendor-8c88cdde22c88a7983ca101bd4964015.js:2150:9)
    at MutationObserver._boundAutorunEnd (serve://dist/assets/vendor-8c88cdde22c88a7983ca101bd4964015.js:2102:575)
(anonymous) @ vendor-8c88cdde22c88a7983ca101bd4964015.js:3332
vendor-8c88cdde22c88a7983ca101bd4964015.js:3470 Uncaught SyntaxError: Unexpected token < in JSON at position 0
    at parse (<anonymous>)
    at y (vendor-8c88cdde22c88a7983ca101bd4964015.js:4060)
    at S (vendor-8c88cdde22c88a7983ca101bd4964015.js:4068)
    at O (vendor-8c88cdde22c88a7983ca101bd4964015.js:4066)
    at e.invoke (vendor-8c88cdde22c88a7983ca101bd4964015.js:2092)
    at e.flush (vendor-8c88cdde22c88a7983ca101bd4964015.js:2085)
    at e.flush (vendor-8c88cdde22c88a7983ca101bd4964015.js:2096)
    at e._end (vendor-8c88cdde22c88a7983ca101bd4964015.js:2150)
    at MutationObserver._boundAutorunEnd (vendor-8c88cdde22c88a7983ca101bd4964015.js:2102)
```

**Comment by devinus** [comment]
@pentastream Can you give me some request/response examples from the `Network` tab?

**Comment by kennedybaird** [comment]
Have same issue on .deb install

I get:
`(nano-wallet-desktop:8118): Gtk-WARNING **: 12:59:04.929: Theme parsing error: gtk.css:68:35: The style property GtkButton:child-displacement-x is deprecated and shouldn't be used anymore. It will be removed in a future version                                       
                                                                                                                                     
(nano-wallet-desktop:8118): Gtk-WARNING **: 12:59:04.929: Theme parsing error: gtk.css:69:35: The style property GtkButton:child-displacement-y is deprecated and shouldn't be used anymore. It will be removed in a future version                                       
                                                                                                                                     
(nano-wallet-desktop:8118): Gtk-WARNING **: 12:59:04.929: Theme parsing error: gtk.css:73:46: The style property GtkScrolledWindow:scrollbars-within-bevel is deprecated and shouldn't be used anymore. It will be removed in a future version                            
[12:59:05.396] [info] Starting application                                                                                           
[12:59:05.425] [info] Deleting outdated assets: config.json, rai_node                                                                
[12:59:05.809] [info] Application window ready to show: Nano Wallet Desktop                                                          
`

And the GUI hangs on the loading animation.

Had same issue with last release.

Ubuntu 18.04

**Comment by stefonarch** [comment]
It looks like a localization issue: setting my desktop to en_US instead of it_IT it is starting.

**Comment by kennedybaird** [comment]
Same, confirmed, now loads. Snappy. Was wondering what the hell was going on.

Changed to en_US from en_NZ and it worked.

**Comment by devinus** [comment]
This should be fixed from v1.0.0.rc.2 and up.
